src/librustc_back/target/mod.rs

```diff
@@ -63,11 +63,12 @@ mod netbsd_base;
 mod solaris_base;
 mod windows_base;
 mod windows_msvc_base;
+mod thumb_base;
 
 pub type TargetResult = Result<Target, String>;
 
 macro_rules! supported_targets {
-    ( $(($triple:expr, $module:ident)),+ ) => (
+    ( $(($triple:expr, $module:ident)),+, ) => (
```

Comment:
You can actually move the comma inside the parens to require a trailing comma like:

```rust
( $(($triple:expr, $module:ident),)+ )
```